### PR TITLE
Minu

### PR DESCRIPTION
**Pull request by Minuja23**
changes made prior to release such as commented codes are removed, codes are formatted, constants and failure status codes checked and organized - and validation checked - (Generally)

### CODE REVIEW

File: defect-tracker-server/src/main/java/com/defect/tracker/services/EmployeeServiceImpl.java [hunk 1]
```diff
@@ -37,49 +33,46 @@ public boolean isEmailAlreadyExist(String email) {
...
-
 	@Override
 	public Optional<Employee> findById(Long id) {
-		
-		return  employeeRepository.findById(id);
+
+		return employeeRepository.findById(id);
 	}
 
 	@Override
```
Comment (Sniththilan) [inline defect-tracker-server/src/main/java/com/defect/tracker/services/EmployeeServiceImpl.java:67]: remove this empty line

File: defect-tracker-server/src/main/java/com/defect/tracker/services/ProjectEmployeeAllocationServiceImpl.java [hunk 2]
```diff
@@ -1,55 +1,54 @@
...
-			projectemployeeallocationRepository.save(projectEmp);
-			
-		}
+	@Override
+	public void update(ProjectEmp projectEmp) {
+		projectemployeeallocationRepository.save(projectEmp);
 
+	}
 
 	@Override
```
Comment (Sniththilan) [inline defect-tracker-server/src/main/java/com/defect/tracker/services/ProjectEmployeeAllocationServiceImpl.java:44]: remove the empty line

File: defect-tracker-server/src/main/java/com/defect/tracker/services/EmployeeServiceImpl.java (hunk 1 above)
Comment (Minuja23) [reply defect-tracker-server/src/main/java/com/defect/tracker/services/EmployeeServiceImpl.java:67]: removed

File: defect-tracker-server/src/main/java/com/defect/tracker/services/ProjectEmployeeAllocationServiceImpl.java (hunk 2 above)
Comment (Minuja23) [reply defect-tracker-server/src/main/java/com/defect/tracker/services/ProjectEmployeeAllocationServiceImpl.java:44]: removed

**Review by Sniththilan — APPROVED** [review]

**Pull request merged**